generator/templates/SDLRPCParameterNames.h

```diff
@@ -0,0 +1,14 @@
+{% include 'copyright.txt' %}
+//  SDLRPCParameterNames.h
+
+#import <Foundation/Foundation.h>
+#import "SDLMacros.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NSString* SDLRPCParameterName SDL_SWIFT_ENUM;
+{% for param in params %}
+extern SDLRPCParameterName const SDLRPCParameterName{{ param.name }};
```

Comment:
This is generating duplicate parameter names in the .h and .m when the data has upper vs. lower case. I don't think that the upper-case ones are needed; or at least I couldn't find where they were used. It seems like they come from struct definitions in the spec, but they're not used anywhere. Can you check on this and see if they're needed? If they are, they'll need to be separated somehow.

Reply:
added uniqueness by `name` in
https://github.com/smartdevicelink/sdl_ios/pull/1556/commits/ede8a54d8f35cbb6a7985a1e04a87a2b842fe5cb